app/helpers/users_helper.rb

```diff
@@ -3,4 +3,10 @@ def user_displayed_attributes
     { name: "Name",
       email: "Email" }
   end
+
+  def previous_application_attributes
+    { track: "Track",
+      admin_status: "Admin status",
+      user_status: "User status" }
+  end
```

Comment:
this should be in `applications_helper.rb` since all of these attributes are on the application now and not the user.